SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/TVA/SOE.tsx

```diff
@@ -57,17 +57,20 @@ const SOE: React.FC<SEBrowser.IWidget<any>> = (props) => {
     }
 
     function HandleStatusFilterChange(key: string) {
-        statusFilter[key] = !statusFilter[key]
+        const lowercaseKey = key.toLowerCase();
+        statusFilter[lowercaseKey] = !statusFilter[lowercaseKey];
         setStatusFilter(statusFilter);
-        BuildTable(soeInfo)
+        BuildTable(soeInfo);
     }
 
     function BuildTable(data) {
-        let tbl = data.filter(si => !statusFilter[si.Status]).map((si, index) => <tr key={index}>
-            <td>{si.Time}</td>
-            <td>{si.Alarm}</td>
-            <td>{si.Status}</td>
-        </tr>)
+        let tbl = data.filter(si => !statusFilter[si.Status.toLowerCase()]).map((si, index) => (
```

Comment:
because the key is still `ABNORMAL` it will throw an error when looking for `abnormal`

Reply:
Thank you for letting me know. I wish I had access to SOE data to test this out.

Reply:
This has been corrected.